crates/sui-tool/src/commands.rs

```diff
@@ -54,11 +54,14 @@ pub enum ToolCommand {
         )]
         validator: Option<AuthorityName>,
 
+        // At least one of genesis or fullnode_rpc must be provided
         #[clap(long = "genesis")]
-        genesis: PathBuf,
+        genesis: Option<PathBuf>,
 
-        #[clap(long = "history", help = "show full history of object")]
-        history: bool,
+        // At least one of genesis or fullnode_rpc must be provided
+        // RPC address to provide the up-to-date committee info
+        #[clap(long = "fullnode-rpc")]
```

Comment:
Maybe useful to provide an example. Also is it better to name this `fullnode-addr`?

Reply:
renamed to `fullnode-rpc-url` to be consistent with other cli tools